react/components/ShippingSimulator/index.js

```diff
@@ -41,6 +41,12 @@ const ShippingSimulator = ({
     setIsValid(postalCode.valid)
   }
 
+  const keyPress = e => {
+    if(e.which == 13){
+      handleClick(e);
+    }
+  }
+
   const handleClick = e => {
     e.preventDefault()
```

Comment:
The prop onSubmit doesn't have an event, so we should check for event before calling `preventDefault`
```suggestion
    e && e.preventDefault()
```